torch/onnx/symbolic_registry.py

```diff
@@ -22,6 +22,17 @@ def register_version(domain, version):
         _registry[(domain, version)] = {}
     register_ops_in_version(domain, version)
 
+def register_quantized_ops(domain, version):
```

Comment:
let's also move this function to symbolic_caffe2.py as well. Right now, it's very specific to caffe2.